Cargo.toml

```diff
@@ -91,4 +91,4 @@ codegen-units = 1
 lto = true
 
 [workspace]
-members = ["ivf", "crates/nasm_rs", "crates/avformat-sys"]
+members = [".", "ivf", "crates/nasm_rs", "crates/avformat-sys"]
```

Comment:
wut

Reply:
It is a shortcoming of the toolchain, cargo-metadata does not have the concept of main crate and cargo-metadata does not put the main crate as first otherwise.

Reply:
have you filed an upstream bug?

Reply:
Yes, but cargo-metadata does not have a mean to provide the information I need.